common/pb/response_ban_history.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
+import "response.proto";
+
+message Response_BanHistory{
+    extend Response {
+        optional Response_BanHistory ext = 1012;
+    }
+    repeated string admin_id = 1;
+    repeated string admin_name = 2;
+    repeated string ban_time = 3;
+    repeated string ban_length = 4;
+    repeated string ban_reason = 5;
+    repeated string visible_reason = 6;
```

Comment:
This is confusing. We should add a new `ServerInfo_Ban` type which contains all of the data, then this message just contains `repeated ServerInfo_Ban` instead of all these fields.